app/models/comment.rb

```diff
@@ -2,9 +2,21 @@ class Comment < ApplicationRecord
   belongs_to :post
   validates_presence_of :body
 
+  def self.best(comments)
+    comments.sort { |x,y| determine_sentiment(y) <=> determine_sentiment(x) }.first
+  end
+
+  def self.rank_order(comments)
```

Comment:
Author: Ya, I did, but after talking to the product people it sounds they are still figuring out how this feature will be extended to other parts of the product. I'd rather not get too too abstract.